src/config.coffee

```diff
@@ -832,25 +832,29 @@ class Config
       @configFileHasErrors = false
     catch error
       @configFileHasErrors = true
-      @notifyFailure('Failed to load config.cson', error)
+      if error.location?
+        @notifyFailure('Failed to load config.cson', error.stack)
```

Comment:
This might be `config.json` so you may want to use `path.basename(@getUserConfigPath())`